### DISCUSSION TITLE
Resume & Stop doesn't appear when I use Creality Cloud. Ender3 v2

### DISCUSSION THREAD

**Discussion by ddavidds** [mriscoc/Ender3V2S1#1357] [Q&A] [answered]
Hi all

Thank you for the firmware it's amazing. I'm testing the last version in the Ender3 v2 with Creality Box 2.0, works nice but when I launch the impression from Creality Cloud through my Creality Box2.0, the Ender begins the work without problem but in the display doesn't appear the pause or stop button. 
I can see the temperatures and position but I need to pause and stop from the cloud. Another interesting thing is when I pause the job, CCloud says "Push continue in the printer" but I don't have button. It resumes automatically if I unplug and plug the wire (microUsb).

Do you know what is happening? 

Thank you in advance

**Comment by mriscoc** [comment, marked as answer]
Hi, for Creality Cloud support, the firmware must be compiled with CCLOUD_PRINT_SUPPORT enabled:

https://github.com/mriscoc/Ender3V2S1/blob/404e13a8a0f58ce08d2c95784a264bed9aea252f/Marlin/Configuration.h#L3451

**Reply by ddavidds** [reply]
Ow thanks
I really appreciate your response.

Thank u so much!
